library/cwm/src/lib/cwm/common_widgets.rb

```diff
@@ -365,6 +365,40 @@ class RichText < AbstractWidget
     self.widget_type = :richtext
 
     include ValueBasedWidget
+
+    # Determines if the scroll should be restored after updating the content
```

Comment:
it is just vertical scrolling now.

Reply:
I'll update those documentation lines. Thanks